components/CaseTable.tsx

```diff
@@ -31,20 +26,23 @@ export const CaseTable: React.FunctionComponent<ICaseTableProps> = (props) => {
             wrap: true,
             sortable: true,
         },
-        {
-            name: 'Primary Diagnosis',
-            selector: 'PrimaryDiagnosis',
-            wrap: true,
-            sortable: true,
-        },
-        {
-            name: 'Age at Diagnosis (years)',
-            selector: 'AgeatDiagnosis',
-            format: (sample: Entity) =>
-                convertAgeInDaysToYears(sample.AgeatDiagnosis),
-            wrap: true,
-            sortable: true,
-        },
+        ...generateColumnsByDataSchema(
+            SchemaDataId.Diagnosis,
```

Comment:
@inodb Do you know if `Diagnosis` is this the correct schema for Cases table?

Reply:
yes, though there could be multiple. But we can improve over time. E.g. there are a few others that might be at the "Cases" level